setup.py

```diff
@@ -41,6 +41,6 @@
     packages=setuptools.find_packages(exclude=["tests"]),
     python_requires=">=3.8",
     url="https://github.com/CellProfiler/core",
-    version="4.2.1-dg",
+    version="4.2.1",
```

Comment:
how do people select this version then? pip from github hash?

Reply:
Yes. Currently trying to get it to work.

Reply:
Verified locally that it works, I had to rename the package itself back to cellprofiler_core so that I don't have to mess with the imports in CellProfiler (CellProfiler imports cellprofiler_core).